sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/AggregationQuerySuite.scala

```diff
@@ -594,7 +594,7 @@ abstract class AggregationQuerySuite extends QueryTest with SQLTestUtils with Te
           |  max(distinct value1)
           |FROM agg2
         """.stripMargin),
-      Row(-60, 70.0, 101.0/9.0, 5.6, 100))
+      Row(-60, 70, 101.0/9.0, 5.6, 100))
```

Comment:
sum of int is long, we shouldn't use double here.

Reply:
uh... checkAnswer was unable to detect this